### ISSUE TITLE
lsp-ui-doc : `q` closes the window AND types "q"

### ISSUE THREAD

**Issue by Trevoke** [emacs-lsp/lsp-ui#608]
This is probably just something dumb I'm doing but when the doc window opens and I hit "q", it does close the doc window... But it also types "q" in the buffer, which isn't something I desire.

Is there a way for me to *only* close the doc pop-up?

**Comment by yyoncho** [comment]
`q` is just `q` and the window is closed because the buffer has changed. It will close on any char. There is `lsp-ui-doc-hide`  which you can use to hide the doc window.

**Comment by Trevoke** [comment]
Ah, OK!

What's `lsp-ui-doc-frame-mode-map` ?

**Comment by yyoncho** [comment]
https://www.gnu.org/software/emacs/manual/html_node/elisp/Keymap-Basics.html

**Comment by Trevoke** [comment]
No... I mean... What is it *for* ? Based on the name I thought it might be for the documentation pop-up but obviously not (it is where I got `q` from)

**Comment by yyoncho** [comment]
I don't understand the question. This object is keymap and it is used to bind keys in the posframe window.

**Comment by Trevoke** [comment]
Okay now I just have to figure out what the postframe window was.

How many of these questions that I am asking should be answered in the README, by the way?

**Comment by yyoncho** [comment]
> Okay now I just have to figure out what the postframe window was.

The posframe is the popup window.

> How many of these questions that I am asking should be answered in the README, by the way?

Definition of what keymap is in emacs docs. The same is true for posframe.

**Comment by Trevoke** [comment]
Okay, so ... How come the button "q" in the postframe window keymap isn't
attached to lsp-ui-doc-hide ?
